File: app/src/main/java/com/arash/altafi/room/java/user_model.java
package com.arash.altafi.room.java;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tbl_user")
public class user_model {

    @PrimaryKey(autoGenerate = true)
    int id;
    @ColumnInfo(name = "user_name")
    String name;
    @ColumnInfo(name = "user_family")
    String family;
    @ColumnInfo(name = "user_age")
    int age;

}

File: app/src/main/java/com/arash/altafi/room/kotlin/UserModel.kt
package com.arash.altafi.room.kotlin

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tbl_user_kotlin")
class UserModel {
    @PrimaryKey(autoGenerate = true)
    var id: Int? = null

    @ColumnInfo(name = "user_name")
    var name: String? = null

    @ColumnInfo(name = "user_family")
    var family: String? = null

    @ColumnInfo(name = "user_age")
    var age: Int? = null
}